### DISCUSSION TITLE
Daemons in a k8s deployment stop sending heartbeats

### DISCUSSION THREAD

**Discussion by atzinier** [dagster-io/dagster#19820] [Q&A]
Hello, I have a dagster deployment in k8s using helm. I have two clusters, one for prod and one for dev. I have been running a schedule which ingests data hourly for both environments. The issue occurs between 07:30-08:00 when some of the daemons stop sending heartbeats. The problem is that the dev environment works as expected. It's only the prod that it's failing.  I have verified that we don't reach any memory limit and I have checked the daemon pod logs in k8s but I don't see something. I have tried to downgrade to the version that I first noticed the problem but the issue is still there. Do you know how could I debug/solve this problem? 
Dagster version: 1.6.3 
Differences between prod and dev:

1. k8s cluster
2. PostgresDB for run logs
3. Snowflake DB for landing the data

<img width="1439" alt="Screenshot 2024-02-15 at 08 26 33" src="https://github.com/dagster-io/dagster/assets/120724158/2a887965-e481-4575-a93e-465f2a8f6935">
<img width="1009" alt="Screenshot 2024-02-15 at 09 29 38" src="https://github.com/dagster-io/dagster/assets/120724158/85ab3e74-3ce8-46de-8d70-26f684f5bec9">

<img width="1004" alt="Screenshot 2024-02-15 at 08 46 30" src="https://github.com/dagster-io/dagster/assets/120724158/e3f45d25-4633-46a0-bb7c-14a7095236d6">

**Comment by shalabhc** [comment]
Hi @atzinier - since you see this at a specific time, to debug further I recommend logging into the daemon pods at the specific time that the heartbeats fail, and checking if you can connect to the database? Maybe there are scheduled network or database issues that prevent daemons from writing their heartbeats?
